tool/update_blns.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:convert';
+import 'dart:io';
+
+final _blnsJsonRawUrl =
+    'https://github.com/minimaxir/big-list-of-naughty-strings/raw/master/blns.json';
+final _blnsFilePath = 'test/blns.dart';
+
+Future<Null> main() async {
```

Comment:
Wonder if it'd be better to just store the JSON file and read from that instead.

Then you don't have to worry that you encoded everything correctly.

Reply:
I'll leave this for now.

Reply:
ack